readme.md

```diff
@@ -1,25 +1,84 @@
-# Continous Compliance
+# Continous Compliance GitHub Action
 
-## What is it
+This GitHub Action will automatically gather compliance information about:
 
-GitHub Action that gather compliance evidence and store it in the cloud
+- Documentation
+- Test results
+- GitHub pull request information and commits associated
 
-## Inputs
+And stored in a Google Cloud Storage bucket for future reference for regulators.
```

Comment:
Might be clearer to say 
"And store it in a Google Cloud Storage bucket as evidence for regulators". ?

Reply:
Thanks